src/shared/components/ResourceList/index.tsx

```diff
@@ -130,7 +133,7 @@ const ResourceListComponent: React.FunctionComponent<{
       <div className="resource-list">
         <h3 className={`header ${busy ? '-fetching' : ''}`}>
           <RenameableItem
-            defaultValue={name}
+            defaultValue={!!name ? name : 'Unnamed List'}
```

Comment:
it can just be `defaultValue={name || 'Unnamed list'}`

Reply:
yes!